models/TradingVenue.py

```diff
@@ -1,36 +1,29 @@
 import os
 from dotenv import load_dotenv
-from helpers import RequestHandler
 from datetime import datetime, timedelta
+from lemon import api
 import datetime
 
+load_dotenv()
+client = api.create(
+    trading_api_token=os.environ.get('TRADING_API_KEY'),
+    market_data_api_token=os.environ.get('DATA_API_KEY'),
+    env='paper'
+)
+print(client)
 
-class TradingVenue(RequestHandler):
 
+class TradingVenue:
     def __init__(self, is_open: bool = False):
         self.is_open = is_open
 
-    def check_if_open(self):
-        load_dotenv()
-        mic = os.getenv("MIC")
-        endpoint = f'venues/?mic={mic}'
-        response = self.get_data_data(endpoint)
-        self.is_open = response['results'][0].get('is_open', None)
-        return self.is_open
-
-    def get_opening_times(self):
-        load_dotenv()
-        mic = os.getenv("MIC")
-        endpoint = f'venues/?mic={mic}'
-        response = self.get_data_data(endpoint)
-        return response
-
     def seconds_till_tv_opens(self):
-        times_venue = self.get_opening_times()
+        load_dotenv()
+        times_venue = client.market_data.venues.get(os.getenv('MIC'))
         today = datetime.datetime.today()
-        opening_days_venue = times_venue['results'][0].get('opening_days', None)
+        opening_days_venue = times_venue.results[0].opening_days
```

Comment:
work with the "get" attribute here as well? This makes error handling/identification easier. Would also be more consistent with the other lines